packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -47,7 +47,9 @@
       }
 
       &:focus {
-        border-color: var(--terra-search-field-button-focus-border-color, #26a2e5);
+        box-shadow: var(--terra-search-field-clear-focus-box-shadow);
+        outline: var(--terra-search-field-focus-keyboard-outline, 2px dashed #000);
+        outline-offset: var(--terra-search-field-focus-keyboard-outline-offset, 1px);
```

Comment:
This pattern is showing up a lot in our code with the accessibility updates.  Should this be converted to a mixin that can be reused or some other reusable concept?

Reply:
Discussed offline with Charles and team: will need to discuss as team the default focus indicators for each theme and go forward strategy to implement in a way that all components will receive a consistent styling